.github/workflows/release-e2e.yaml

```diff
@@ -0,0 +1,57 @@
+---
+name: release-e2e
+
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
```

Comment:
Should this be on tag pushes rather than release creation?

Also, what's the purpose of running this on workflow_dispatch events?

Reply:
> Should this be on tag pushes rather than release creation?

nvmd, I see that we need the release to exist so that we can use the release `rukpak.yaml` manifest.

Reply:
Yeah, we can probably drop the worklow_dispatch event trigger here.

Reply:
Will drop. My apologies. That's debugging cruft I forgot to remove so I could trigger it manually for testing.